packages/contracts/contracts/protocol/Exchange/MixinTransactions.sol

```diff
@@ -16,6 +16,7 @@
 
 */
 pragma solidity 0.4.24;
+pragma experimental ABIEncoderV2;
```

Comment:
I believe this will actually change the bytecode of this contract and we purposely removed `pragma experimental ABIEncoderV2;` when not necessary because it reduced gas costs. What is the reason for adding this back in?

Edit: I see why we need this to generate public wrappers for all of these internal functions. Is there a better way to do this that doesn't require us altering the underlying contract?

Reply:
If the final `Exchange` contract has ABIv2 enabled, doesn't that automatically enable it for all parent contracts?

There are ways around it, I just thought it was incorrectly missing here since everything will be compiled with ABIv2 anyway.